src/main/java/org/veupathdb/service/sequence/service/Sequences.java

```diff
@@ -0,0 +1,54 @@
+package org.veupathdb.service.sequence.service;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import org.veupathdb.service.sequence.model.Config;
+import org.veupathdb.service.sequence.model.ReferenceSequenceSpec;
+
+import htsjdk.samtools.reference.IndexedFastaSequenceFile;
+import htsjdk.samtools.reference.FastaSequenceIndex;
+
+import java.nio.file.Paths;
+import java.nio.file.Path;
+import java.util.Map;
+import java.util.HashMap;
+import java.util.Objects;
+
+public class Sequences {
+  private static final Logger LOG = LogManager.getLogger(Sequences.class);
+
+  private static Map<String, ReferenceSequenceSpec> sequenceSpecs;
+  private static Map<String, FastaSequenceIndex> sequenceIndexes;
+  private static Map<String, IndexedFastaSequenceFile> sequenceFiles;
```

Comment:
> Do we know how many different sequences we will need to support retrieval for?
yes, about five!

> I imagine if it's sufficiently many we may not be able to hold everything in memory at once.
I agree. I've made the reference sequences configurable through `config.json` so that the service can be horizontally scaled, but picking the right amount of memory for the JVM so that it can hold them in memory is crucial.